src/test/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterControllerTest.java

```diff
@@ -30,13 +35,26 @@ public void should_return_401_when_auth_token_is_invalid() throws Exception {
         given(validator.getServiceName(anyString()))
             .willThrow(InvalidTokenException.class);
 
-        mockMvc.perform(
+        sendLetter()
+            .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    public void should_return_400_if_letter_was_already_set() throws Exception {
+        doThrow(LetterAlreadySentException.class)
```

Comment:
IMHO 400 Error code should be used only when request syntax is not correct.what you think about status code 422?

Reply:
`400` is currently used more widely.
In [RFC266](https://tools.ietf.org/html/rfc2616#section-10.4.1) it was
>The request could not be understood by the server due to malformed
   syntax. The client SHOULD NOT repeat the request without
   modifications.

But nowadays ([RFC 7231](https://tools.ietf.org/html/rfc7231#section-6.5.1)):

>the server cannot or will not process the request due to something that is perceived to be
   a client error

Reply:
HMCTS status codes guide:
https://hmcts.github.io/restful-api-standards/#150

Reply:
ok RFC 7231 states examples for malformed request syntax, invalid request
message framing, or deceptive request routing.If you see Github it handles duplicate repository creation by sending back error code 422.Also status code 422 seems to be missing in HMCTS status code.I think it's fine for now 👍

Reply:
422 is not in the RFC either
https://tools.ietf.org/html/rfc7231#section-6.5